packages/css/src/components/logo/logo.scss

```diff
@@ -6,6 +6,7 @@
 .ams-logo {
   block-size: var(--ams-logo-block-size);
   display: block;
+  min-block-size: var(--ams-logo-min-block-size);
```

Comment:
Shall we try a container query to show just the crosses on narrow screens?

Reply:
Sounds interesting, but what would you want the container query to test that a ‘regular’ window width query can’t? Is it to consider the presence of an app title?

Reply:
The header has a max width so it would make sense I suppose. You could use a `:has()` selector in combination with a container (or normal) query to hide the logo text. So probably not in this file but that defies the purpose of having isolated components, so targeting logo in the header CSS is wrong but needed? That's probably why it should be a pattern and not a component?